src/wp-includes/pluggable.php

```diff
@@ -2676,11 +2676,9 @@ function wp_hash_password(
 		 * - `PASSWORD_ARGON2ID`
 		 * - `PASSWORD_DEFAULT`
 		 *
-		 * The values of the algorithm constants are strings in PHP 7.4+ and integers in PHP 7.3 and earlier.
-		 *
 		 * @since 6.8.0
 		 *
```

Comment:
```suggestion
		 * @since 6.9.0 The paramater passed will always be a string.
		 *
```

Document that the filter is changing even though this is a very minor change